code/game/objects/items/devices/lightreplacer.dm

```diff
@@ -154,12 +154,16 @@
 		user.balloon_alert(user, "lights inserted")
 		return TRUE
 
-/obj/item/lightreplacer/emag_act()
+/obj/item/lightreplacer/emag_act(mob/user, obj/item/card/emag/emag_card)
 	if(obj_flags & EMAGGED)
-		return
+		return TRUE
 	obj_flags |= EMAGGED
 	playsound(loc, SFX_SPARKS, 100, TRUE, SHORT_RANGE_SOUND_EXTRARANGE)
 	update_appearance()
+	if (user)
+		balloon_alert(user, "gas synthesizer scrambled")
+		to_chat(user, span_warning("[src]'s lights are now filled with plasma! Be careful to only install them in disabled light fixtures, lest they explode!"))
```

Comment:
This is a good place for a `to_chat` but the sound effect makes the accompanying `balloon_alert` unecessary.
Which also makes the check for `user` unecessary as `to_chat` can handle being passed a null, oh well.